frontend/src/js/tooltip/Tooltip.tsx

```diff
@@ -55,7 +57,7 @@ const TackIconButton = styled(IconButton)`
   margin-left: 5px;
 `;
 const TypeIcon = styled(StyledFaIcon)`
-  margin-right: 10px;
+  margin-right: 5px;
```

Comment:
Nice, I think that's more similar to what it looks like in the concept trees as well, exact would probably be
```suggestion
  margin-right: 6px;
```